docs/marketplace/dev/front-end/ui-components-library/cache/static.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Cache Strategy Static
+description: This article provides details about the Cache Strategy Static service in the Components Library.
+template: concept-topic-template
+---
+
+This article provides details about the Cache Strategy Static service in the Components Library.
+
+## Overview
+
+Cache Strategy Static is an Angular Service that adds values immediately to the 
+cache until the expiration date and always retrieves them from cache if requested.
+See an example below, how to use the Cache Strategy Static service.
+
+`type` - a cache type.  
+`expiresIn` - represents a duration as string where components are separated by a space.  
+
+```html
+<spy-select
+  [datasource]="{
+    type: 'http',
+    ...,
+    cache: {
+      type: 'static',
+      expiresIn: '1m 10d 3h',
```

Comment:
It is not clear how the minutes should be added.

Reply:
Added examples to the interface